volontulo/templates/offers/offer_apply.html

```diff
@@ -14,7 +14,7 @@
             </div>
             <label class="col-xs-1 control-label" for="{{ form.email.id_for_label }}">Email:</label>
             <div class="col-xs-4">
-               <input type="text" value="{{ request.POST.email|default:volunteer_user.user.email }}" name="email" class="form-control" id="{{form.email.id_for_label }}"{% if user.is_authenticated %}disabled="disabled"{% endif %} />
```

Comment:
Well, I'm not talking about minifying code, where variables comes shorter or something like this. I'm talking about code that is clear, easy to understand and leaving no room for speculation (like my previous comment about value of this attribute).
Another reason for using additional attributes (and self-closing one tag elements) is that we can use this document as proper XML data.
I'll recall Perl language. We often could write something like `while(<>) {}` to make a loop, but it doesn't mean it's a good idea, because you MUST know that it means `while (defined($_ = <>)) {...}`.
Less code not always mean better.